conftest.py

```diff
@@ -21,6 +21,7 @@
     'pytest_plugins.requirements.update_requirements',
     'pytest_plugins.sanity_plugin',
     'pytest_plugins.video_cleanup',
+    'pytest_plugins.capsule_n-minus',
```

Comment:
To simplify this naming?
```suggestion
    'pytest_plugins.capsule_nminus',
```

Reply:
May be newbie wont understand it quick. Also there is no standard in naming convention. Skipping it as non-blocking.